### PR TITLE
New package: cava-gui-0.5.0

### PR DESCRIPTION
**Pull request by nikp123**

### CODE REVIEW

**Comment by Johnnynator** [comment]
The template is a bit messy (and not following any rules and guidelines as how a template should be)
You probably can copy the [cava template](https://github.com/voidlinux/void-packages/blob/master/srcpkgs/cava/template) over and edit the required bits.

**Comment by nikp123** [comment]
Apologize for that, forgot to exclude architectures.

**Comment by nikp123** [comment]
@Johnnynator Can I change the directory in which it automaticly builds it in.
It tries to cd into cava-gui while cava is the right folder.

**Comment by Johnnynator** [comment]
@nikp123 yep set $wrksrc to cava-${version}

**Comment by nikp123** [comment]
@Johnnynator thanks, now it works

File: srcpkgs/cava-gui/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'Cava'
+pkgname=cava-gui
+version=0.5.0
+revision=1
+_pkgname=cava
+only_for_archs="i686 x86_64"
```
Comment (Johnnynator) [inline srcpkgs/cava-gui/template]: only_for_archs should only be used if the program is designed to run only on these platform.
It normally should cross-compile fine with the current template, so just remove this line.

**Comment by Johnnynator** [comment]
You should rebase it so it is only one commit and not 4.

**Comment by nikp123** [comment]
@Johnnynator how do I do that? I just don't want risking breaking the whole repo.

**Comment by Johnnynator** [comment]
you can e.g run `git rebase -i HEAD~4 ` and then fixup or squash commits.

**Comment by nikp123** [comment]
@Johnnynator thanks again :)

**Comment by Vaelatern** [comment]
The template is decent. I'd recommend the following patch:
```
diff --git a/srcpkgs/cava-gui/template b/srcpkgs/cava-gui/template
index 39a6ebacc0..320f69a763 100644
--- a/srcpkgs/cava-gui/template
+++ b/srcpkgs/cava-gui/template
@@ -1,19 +1,19 @@
-# Template file for 'Cava'
+# Template file for 'cava-gui'
 pkgname=cava-gui
 version=0.5.0
 revision=1
-_pkgname=cava
+wrksrc="cava-${version}"
 build_style=gnu-configure
 hostmakedepends="automake libtool"
-makedepends="alsa-lib-devel pulseaudio-devel fftw-devel ncurses-devel iniparser-devel SDL2-devel libX11-devel"
+makedepends="alsa-lib-devel pulseaudio-devel fftw-devel ncurses-devel
+ iniparser-devel SDL2-devel libX11-devel"
 short_desc="Console-based Audio Visualizer for ALSA (GUI branch)"
 maintainer="nik123 <pavlica.nikola@gmail.com>"
 homepage="https://github.com/nikp123/cava"
 license="MIT"
-distfiles="https://github.com/nikp123/${_pkgname}/archive/${version}.tar.gz"
+distfiles="https://github.com/nikp123/cava/archive/${version}.tar.gz"
 checksum="4a017b2445edcd6338fcd2aeb3ab911f077ca2bf49b3f5c5d5453e6576d9dd00"
 conflicts=cava
-wrksrc="cava-${version}"

 pre_configure() {
        NOCONFIGURE=1 ./autogen.sh
```
@voidlinux/pkg-committers is this a template we want? My concern is it seems to be a fork of a maintainer project, and essentially a flavor.

**Comment by lemmi** [comment]
@voidlinux/pkg-committers @nikp123 seems to me the original project is not dead yet. Is there any chance that
cava-gui can be merged back to cava?

**Comment by nikp123** [comment]
@lemmi Hardly. I tried multiple times to merge, seems like my code needs to be tested a lot before merging, plus I will probably break someone else's pull request because of the huge code difference (about 50KiB). So I will maintain this fork until it's ready to merge.

**Comment by nikp123** [comment]
@voidlinux/pkg-commiters @Vaelatern @lemmi Patched and rebased.

**Comment by lemmi** [comment]
@nikp123 If you feel this is more or less a development fork for cava, I'd consider not merging it. We prefer having mostly stable software in the repo. In case you think this is stable and a worthy addition, we can go ahead an merge this.

**Comment by nikp123** [comment]
@lemmi Ok, then it will need to wait a while. Since it is constantly updated with features, I'm not sure about the stability part.
Thanks for the notice, I don't want to ruin this repo accidentally.

**Comment by lemmi** [comment]
@nikp123 Thanks for the honesty. I'll close this PR then. Feel free to open a new one when your fork is stable or merged back.

**Pull request closed without merging**